etc/taskcluster/docker/base.dockerfile

```diff
@@ -41,4 +41,6 @@ RUN \
     # Python 2 bits that have been removed from Ubuntu packages
     curl https://bootstrap.pypa.io/2.7/get-pip.py -sSf -o get-pip.py && \
     python2 get-pip.py && \
-    python2 -m pip install virtualenv
+    python2 -m pip install virtualenv \
+    # Ensure modern pip is present.
+    python3 -m pip install -U pip
```

Comment:
Should this have been python2?

Reply:
Nope, the relevant build failures happen underneath python3.

Reply:
👍 saw the line above and assumed it must have been a transposition